### PR TITLE
Splice Proposal for Hyperledger Labs

### PR DESCRIPTION
**Pull request by waynecollier-da**
Reference applications for funding, operating, and incentivizing the use of a decentralized, public Canton synchronizer. Includes the Amulet reference application for creating native payment utilities for Canton synchronizers and Daml applications.

### CODE REVIEW

**Review by tkuhrt — CHANGES_REQUESTED** [review]
Thanks for the submission. I agree to be a sponsor for this lab. One minor change, and then I will approve.

File: labs/images/Super Validator Architecture Diagram for Hyperledger Splice-1.jpg
Comment (tkuhrt) [inline labs/images/Super Validator Architecture Diagram for Hyperledger Splice-1.jpg:1]: Can we change name of the diagram too? Also, it looks like the diagram did not update to remove the Hyperledger?

File: labs/images/Super Validator Architecture Diagram for Hyperledger Splice-2.jpg
Comment (tkuhrt) [inline labs/images/Super Validator Architecture Diagram for Hyperledger Splice-2.jpg:1]: Similar comment to the other diagram.

File: labs/splice.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+layout: default
+title: Splice
+parent: Labs
+---
+# Hyperledger Splice
```
[the remark this answers is not in the document]
Comment (waynecollier-da) [reply labs/splice.md]: Ok will fix.

File: labs/images/Super Validator Architecture Diagram for Hyperledger Splice-1.jpg
Comment (waynecollier-da) [reply labs/images/Super Validator Architecture Diagram for Hyperledger Splice-1.jpg:1]: Yep, will fix

**Comment by waynecollier-da** [comment]
Hi, @tkuhrt I think all the fixes are done now

**Review by tkuhrt — APPROVED** [review]

**Review by knagware9 — APPROVED** [review]
I appove this lab.

**Review by nidhi-singh02 — APPROVED** [review]
Looks great!

**Pull request merged**